modules/uid20IdSystem.md

```diff
@@ -0,0 +1,32 @@
+## UID 2.0 User ID Submodule
+
+UID 2.0 ID Module.
+
+### Prebid Params
+
+Individual params may be set for the UID 2.0 Submodule. At least one identifier must be set in the params.
+
+```
+pbjs.setConfig({
+    usersync: {
+        userIds: [{
+            name: 'uid20',
+            storage: {
+                name: 'uid20id',
+                type: 'html5'
```

Comment:
if it has to be a cookie, why is html5 an option here? publishers can copy the cookie into local storage?

Reply:
This module allows to use either cookie or html5. The value of the id is read from a specific cookie (__uid2_advertising_token) which is not based out the configuration.

Reply:
per my above comments, `storage` config isn't necessary since you're pulling from your own cookie